flexbox/src/main/java/com/google/android/flexbox/FlexboxHelper.java

```diff
@@ -421,10 +421,6 @@ private FlexLinesResult calculateFlexLines(int mainMeasureSpec, int crossMeasure
 
             FlexItem flexItem = (FlexItem) child.getLayoutParams();
 
-            if (flexItem.getAlignSelf() == AlignItems.STRETCH) {
```

Comment:
This condition had to be removed down. If not, the first flex item will be "inside" the indices of the previous flex line.